sweepai/core/vector_db.py

```diff
@@ -208,6 +208,8 @@ def get_deeplake_vs_from_repo(
            and all(not file.endswith(ext) for ext in sweep_config.exclude_exts)
            and all(not file[len("repo/"):].startswith(dir_name) for dir_name in sweep_config.exclude_dirs)
     ]
+    if len(file_list) > 500:
```

Comment:
you need to move this to the top, and set a special boolean. Don't entirely turn off the cache in this case